src/google/protobuf/compiler/cpp/cpp_message.cc

```diff
@@ -1092,7 +1092,7 @@ void MessageGenerator::GenerateClassDefinition(io::Printer* printer) {
 
   format(
       "inline $classname$() : $classname$(nullptr) {}\n"
-      "virtual ~$classname$();\n"
+      "virtual ~$classname$() override;\n"
```

Comment:
I think as long as we're adding `override` we can drop the `virtual` keyword since that is implied. One other thing is would you mind running `./generate_descriptor_proto.sh`? That will rebuild the checked-in pb.h and pb.cc files.